src/browser/BrowserService.cpp

```diff
@@ -315,36 +315,36 @@ QString BrowserService::getCurrentTotp(const QString& uuid)
 }
 
 QJsonArray
-BrowserService::findEntries(const EntryParameters& entryParameters, const StringPairList& keyList, const bool httpAuth)
+BrowserService::findEntries(const EntryParameters& entryParameters, const StringPairList& keyList, bool* accepted)
```

Comment:
Btw, maybe we should cleanup other functions like that also? `KeePass2RandomStream::randomBytes()` as a good example. None of those are checking a null pointer.

Reply:
Yikes, we should. Nothing has exploded.. yet.. since those are internal functions. I don't take any chances with Browser since it is handling user data and could be prime for injection attacks.